cloudbuild.yaml

```diff
@@ -56,6 +74,12 @@ steps:
   - cd e2e-tests;
     docker build -t ${_REPOSITORY}/e2e-tests:${_TAG} .;
     docker build -t ${_REPOSITORY}/e2e-tests/controller-spark:${_TAG} ./controller-spark/.;
+  waitFor: ['-']
+
+- name: 'ubuntu'
+  id: 'Sleep for servers to be up'
+  args: ['sleep', '90']
```

Comment:
Question: Is the 90 seconds wait really needed? I mean what does happen if we remove this and change the `waitFor` of the next step to depend on previous servers? If this step is required, it is an indication that maybe we should add proper `healthcheck` to previous services that we bring up. Otherwise this may add to our e2e run flakiness.

Reply:
This has been replaced with a step `Wait for Initial servers start` that waits for the services to be up with proper health checks